example/CMakeLists.txt

```
@@ -1,11 +1,5 @@
 # Find Kokkos
-find_package(Kokkos QUIET)
-if(NOT Kokkos_FOUND)
-  message(FATAL_ERROR "Kokkos not found, set Kokkos_ROOT properly (current Kokkos_ROOT=${Kokkos_ROOT})")
-endif()
-foreach(_i "1;2;3") # cut .../lib/cmake/Kokkos suffix
-cmake_path(GET Kokkos_DIR PARENT_PATH Kokkos_DIR)
-endforeach()
+find_package(Kokkos REQUIRED)
 message(STATUS "Found installed Kokkos at ${Kokkos_DIR}")
```

We already try to find `Kokkos` earlier and print the path there. I think duplicate that logic here instead of printing the path to the directory containing the configuration file
```suggestion
get_property(Kokkos_INSTALL_DIR TARGET Kokkos::kokkoscore PROPERTY INTERFACE_INCLUDE_DIRECTORIES)
cmake_path(GET Kokkos_INSTALL_DIR PARENT_PATH Kokkos_INSTALL_DIR)
message(STATUS "Found installed Kokkos at ${Kokkos_INSTALL_DIR}")
```

---

I don't see an obvious benefit to do so and your suggestion add complexity.